package.json

```diff
@@ -122,6 +122,8 @@
     "pack-n-play": "^2.0.0",
     "proxyquire": "^2.1.3",
     "sinon": "^18.0.0",
+    "nise": "6.0.0",
+    "path-to-regexp": "6.2.2",
```

Comment:
Let's just add a tacking issue to remove this once we upgrade to Node 18 minimum so we don't forget.